Duplicate identifier 'stack' with TS 1.9

Hi,
Typescript `1.9` (currently `typescript@next`) adds definitions for [`Error.prototype.stack`](https://github.com/nojvek/TypeScript/blob/94d8888b09b1c08737b6311d50335c8c045f7da0/src/lib/es5.d.ts#L867).

It results in [a duplicate identifier](https://github.com/typed-typings/env-node/blob/master/4/node.d.ts#L28)

````text
C:/Users/Charles/AppData/Roaming/npm/node_modules/typescript/lib/lib.d.ts(883,5): error TS2300: Duplicate identifier 'stack'.
typings/globals/node/index.d.ts(19,3): error TS2300: Duplicate identifier 'stack'.
````

How do you plan to handle this sort of changes ? Should we remove `stack` from this definition once TS 1.9 is stable ?

--- a/0.12/node.d.ts
+++ b/0.12/node.d.ts
@@ -9,10 +9,36 @@
 *                                               *
 ************************************************/
 
-interface Error {
-    stack?: string;
-}
+interface NodeError {
+  /**
+   * Returns a string describing the point in the code at which the Error was instantiated.
+   *
+   * For example:
+   *
+   * ```
+   * Error: Things keep happening!
+   *    at /home/gbusey/file.js:525:2
+   *    at Frobnicator.refrobulate (/home/gbusey/business-logic.js:424:21)
+   *    at Actor.<anonymous> (/home/gbusey/actors.js:400:8)
+   *    at increaseSynergy (/home/gbusey/actors.js:701:6)
+   * ```
+   *
+   * The first line is formatted as <error class name>: <error message>, and is followed by a series of stack frames (each line beginning with "at "). Each frame describes a call site within the code that lead to the error being generated. V8 attempts to display a name for each function (by variable name, function name, or object method name), but occasionally it will not be able to find a suitable name. If V8 cannot determine a name for the function, only location information will be displayed for that frame. Otherwise, the determined function name will be displayed with location information appended in parentheses.
+   */
+  stack?: string;
 
+  /**
+   * Returns the string description of error as set by calling new Error(message). The message passed to the constructor will also appear in the first line of the stack trace of the Error, however changing this property after the Error object is created may not change the first line of the stack trace.
+   *
+   * ```
+   * const err = new Error('The message');
+   * console.log(err.message);
+   * // Prints: The message
+   * ```
+   */
+  message: string;
+}
+interface Error extends NodeError {}
 
 // compat for TypeScript 1.5.3
 // if you use with --target es3 or --target es5 and use below definitions,

--- a/4/node.d.ts
+++ b/4/node.d.ts
@@ -9,7 +9,7 @@
 *                                               *
 ************************************************/
 
-interface Error {
+interface NodeError {
   /**
    * Returns a string describing the point in the code at which the Error was instantiated.
    *
@@ -36,8 +36,9 @@ interface Error {
    * // Prints: The message
    * ```
    */
-  // message: string;
+  message: string;
 }
+interface Error extends NodeError {}
 
 interface ErrorConstructor {
   /**

--- a/6/node.d.ts
+++ b/6/node.d.ts
@@ -9,7 +9,7 @@
 *                                               *
 ************************************************/
 
-interface Error {
+interface NodeError {
   /**
    * Returns a string describing the point in the code at which the Error was instantiated.
    *
@@ -36,8 +36,9 @@ interface Error {
    * // Prints: The message
    * ```
    */
-  // message: string;
+  message: string;
 }
+interface Error extends NodeError {}
 
 interface ErrorConstructor {
   /**

